internal/cmd/rsl/remote/check/check.go

```diff
@@ -12,13 +12,13 @@ import (
 type options struct {
 }
 
-func (o *options) Run(cmd *cobra.Command, args []string) error {
+func (o *options) Run(_ *cobra.Command, args []string) error {
 	repo, err := repository.LoadRepository()
 	if err != nil {
 		return err
 	}
 
-	hasUpdates, hasDiverged, err := repo.CheckRemoteRSLForUpdates(cmd.Context(), args[0])
+	hasUpdates, hasDiverged, err := repo.CheckRemoteRSLForUpdates(args[0])
```

Comment:
fine if you want to punt on this for another PR, but we should look to plumb the context through so we can respect cancelled contexts / timeouts.

Reply:
Got it, I'll add it back in and just `_` for now.